source/reference/cross-references.txt

```diff
@@ -0,0 +1,104 @@
+.. _style-guide-cross-references:
+
+================
+Cross References
+================
+
+.. default-domain:: mongodb
+
+Use cross references to link to other sections within the MongoDB documentation.
+You can reference to a specific page, or to a specified reference target on a
+page.
+
+Reference Targets
+-----------------
+
```

Comment:
Suggestion: I think you should port the cross-referenced hyperlink section to here since the steps are listed there. And then from the hyperlinks page, link back to this section here. You can keep your example, or use the one that is currently on the hyperlinks page.

I also considered linking to hyperlinks, but feared it would be repetitive information and may be annoying for readers to jump from page to page. It overall seems to make more sense here.